cmd/report/parameters.go

```diff
@@ -63,19 +63,25 @@ func fetchAndDisplayParameters(cfg config.ServiceTitan, categoryID, reportID str
 
 	paramTable := tablewriter.NewWriter(os.Stdout)
 	paramTable.SetRowLine(true)
-	paramTable.SetHeader([]string{"Paramter name", "Label", "Data type", "Array?", "Required?"})
+	paramTable.SetHeader([]string{"Paramter name", "Label", "Data type", "Array?", "Required?", "Accepted Values"})
 
 	fieldTable := tablewriter.NewWriter(os.Stdout)
 	fieldTable.SetRowLine(true)
 	fieldTable.SetHeader([]string{"Field Name", "Label", "Type"})
 
 	for _, param := range report.Parameters {
+		valuesStr := ""
+		for _, group := range param.AcceptedValues.Values {
+			valuesStr += group[1] + " - " + group[0] + "\n"
+		}
```

Comment:
Even though we assume there should be 2 items in the inner items - we probably as a fallback check if there is only a single item or none - that we print as such - what do you think ? Otherwise this would just panic maybe something like;


```golang
args = []string{}

switch len(group) {
 case 0:
    //skip there is none
 case 1:
   args = append(args, group[0])
 case 2:
   args = append(args, group[1] + " - " + group[0])
}


Then later on use strings.Join(args, ",")
```